Serializer/JMSHandlerRegistry.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Serializer;
+
+use FOS\RestBundle\Context\Context;
+use JMS\Serializer\Context as JMSContext;
+use JMS\Serializer\DeserializationContext as JMSDeserializationContext;
+use JMS\Serializer\SerializationContext as JMSSerializationContext;
+use JMS\Serializer\SerializerInterface;
+use JMS\Serializer\Handler\SubscribingHandlerInterface;
+use JMS\Serializer\Handler\HandlerRegistryInterface;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
+ * @internal do not depend on this class directly.
+ */
+class JMSHandlerRegistry implements HandlerRegistryInterface
```

Comment:
Should be proposed directly to the JMSSerializer but done in FOSRestBundle for the moment to be able to use it without changing our version constraints.